xcm/xcm_token_locations.json

```diff
@@ -243,22 +463,218 @@
+              "parachain": 2001
+            },
+            {
+              "generalKeyV1": "0x0104"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "KUSD",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2000
+            },
+           {
+              "generalKeyV1": "0x0081"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "SDN",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2007
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "XRT",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2048
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "KMA",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2084
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "KBTC",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2092
+            },
+            {
+              "generalKeyV1": "0x000b"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "CSM",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2012
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "NEER",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2096
+            },
+            {
+              "generalKeyV1": "0x000000000000000000"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "AIR",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2088
+            },
+            {
+              "generalKeyV1": "0x0001"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "HKO",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2085
+            },
+            {
+              "generalKeyV1": "0x484b4f"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "TUR",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2114
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "QTZ",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2095
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "CRAB",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2105
             },
             {
-              "GeneralIndex": 0
+              "palletInstance": 5
             }
           ]
         }
```

Comment:
Everything is good. Let's just check again that all tokens in Kusama ecosystem have `v3` version for `interiors` block

Reply:
Fixed